backend/tests/Feature/UserPermissionsTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Models\User;
+use App\Models\Role;
+
+class UserPermissionsTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /**
+     * @return void
+     */
+    public function testCreationOfRandomRole()
+    {
+        $role = Role::factory()->create();
+        $this->assertNotNull($role->name);
+        $this->assertNotEmpty($role->name);
+        $this->assertIsString($role->name);
+    }
+
+    /**
+     * @return void
+     */
+    public function testCreationOfApplicationAdministratorRole()
+    {
+        $name = 'Application Administrator';
+        $role = Role::factory()->create([
+            'name' => $name
+        ]);
+        $this->assertNotNull($role->name);
+        $this->assertNotEmpty($role->name);
+        $this->assertIsString($role->name);
```

Comment:
```suggestion
```
These are all implied by the assertEquals below.. which is what you really want to test.